ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -330,26 +330,6 @@ Click Action From Actions Menu
     Wait Until Page Contains Element       xpath=//tr[td[@data-label="Name"]//*[text()="${item_title}"]]/td[@class="pf-c-table__action"]/div/ul/li/button[text()="${action} ${item_type}"]
     Click Element       xpath=//tr[td[@data-label="Name"]//*[text()="${item_title}"]]/td[@class="pf-c-table__action"]/div/ul/li/button[text()="${action} ${item_type}"]
 
-Handle Deletion Confirmation Modal
```

Comment:
`Handle Deletion Confirmation Modal` was moved to another resource file. The jenkins jobs are no longer available. Did you run all tests depending on this keyword?

Reply:
Yeap